lib/geo/wkb.ex

```diff
@@ -79,7 +81,7 @@ defmodule Geo.WKB do
   @doc """
   Takes a WKB string and returns a Geometry.
   """
-  @spec decode!(binary | iodata()) :: Geo.geometry() | no_return
+  @spec decode!(iodata()) :: Geo.geometry()
```

Comment:
I really don't think there's a use case for accepting an iodata, I'm happy to send a PR which removes the `IO.iodata_to_binary` call and thus this function just accepts a binary. And if there's a use case, we could always add it then. WDYT?

Reply:
Removing the `IO.iodata_to_binary` call makes sense to me. I can see by updating the postgrex extension it's not needed there. I can accept a PR to remove that 👍🏿